matrix/common/constants.py

```diff
@@ -155,3 +162,99 @@ class BundleType(Enum):
         );
     """
 }
+
+# Map from internal matrix service column names to the names used in the
+# project tsv and hence the API surface.
+TABLE_COLUMN_TO_METADATA_FIELD = {
```

Comment:
Is this intended to have a key for every column in Redshift (to define `FIELD_DETAIL`/`FILTER_DETAIL`? If so, it's missing some for example from `publication` and `contributor` tables.

Reply:
Nevermind, I see the queries in `query_constructor` don't include these tables.